sdk/include/ndk/extypes.h

```diff
@@ -1460,6 +1461,14 @@ typedef struct _SYSTEM_BOOT_ENVIRONMENT_V1
 
 // FIXME: Class 91-97
 
+// Class 98
+#if (NTDDI_VERSION >= NTDDI_LONGHORN)
```

Comment:
Alternatively, `NTDDI_VISTA` in case it wasn't present in "longhorn" (aka the alpha/beta of Vista).

Reply:
Seems like it was added a bit later (probably Vista SP1? It is present in SP2). Changed it to `NTDDI_VISTA`.